docs/metrics.md

```diff
@@ -0,0 +1,142 @@
+
+# Tab Split Metrics
+
+Metrics collections and analysis plan for Tab Split as part of the [Firefox Test Pilot program](https://testpilot.firefox.com).
+
+## Analysis
+
+Data collected in this experiment will be used to answer the following high-level questions:
+
+* How often do users load a non-homepage in the sidebar?
+* Are people who use Tab Split heavy tab users?
+
+
+
+## Collection
+Data will be collected with Google Analytics and follow [Test Pilot standards](https://github.com/mozilla/testpilot/blob/master/docs/experiments/ga.md) for reporting.
+
+## Custom Metrics
+*none*
+
+### Custom Dimensions
+
+* `cd1` - The width of the sidebar in pixels.  An integer
+* `cd2` - The number of open tabs in the current window.  An integer
+* `cd3` - The site type requested.  One of `desktop` or `mobile`
+
+### Events
+
+#### `Interface`
+
+##### When the user chooses to load a link in the sidebar from the context menu
```

Comment:
There's three context menu locations: from a page (e.g., clicking on the page background), on a link, or on the tab itself. We have a bug for doing it from a bookmark as well.

Reply:
an event value has to be an integer, so I'll just expand this out into 4 events. thanks